### PR TITLE
Added show puzzle command.

### PR DESCRIPTION
**Pull request by mjlewis-millersville**
Fist implementation of show puzzle. Please check if I got all the edge cases and no bugs. I'd also like to implement color coding, but I don't know how, but I think one of you do, so maybe open another branch and add it to the function later.

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -117,6 +117,13 @@ vorpal
     callback();
   });
```
Comment (JSteve0) [inline src/app.js:119]: Add a comment describing what this section is

**Review by JSteve0 — CHANGES_REQUESTED** [review]
Add documentation to the vorpal command

**Comment by mjlewis-millersville** [comment]
Comments are added!!

**Review by JSteve0 — APPROVED** [review]
Looks great!

**Pull request merged**